pkg/helpers/image/helpers.go

```diff
@@ -148,7 +148,7 @@ func GetRegistryAuthConfigPreference() RegistryAuthConfigPreference {
 		result = RegistryAuthConfigPreference(authPreference)
 	} else {
 		// TODO: remove once deprecated in 4.12
-		klog.Warningln("Defaulting of registry auth file to \"${HOME}/.docker/config.json\" is deprecated. The default will be switched to podman config locations in the future version.")
+		fmt.Fprintf(os.Stderr, "Warning: the default reading order of registry auth file will be changed from \"${HOME}/.docker/config.json\" to podman registry config locations in the future version. \"${HOME}/.docker/config.json\" is deprecated, but can still be used for storing credentials as a fallback.\n")
```

Comment:
Can you add more info (maybe a link) to what the podman registry order means?

Reply:
@soltysh Yup, added a link